deploy-manage/upgrade/deployment-or-cluster/saved-object-migrations.md

```diff
@@ -12,41 +12,40 @@ Each time you upgrade {{kib}}, an upgrade migration is performed to ensure that
 ::::
 
 
-::::{warning} 
-{{kib}} 7.12.0 and later uses a new migration process and index naming scheme. Before you upgrade, read the documentation for your version of {{kib}}.
-::::
+% ::::{warning} 
+% {{kib}} 7.12.0 and later uses a new migration process and index naming scheme. % Before you upgrade, read the documentation for your version of {{kib}}.
+% ::::
 
 
 ::::{warning} 
-The `kibana.index` and `xpack.tasks.index` configuration settings are obsolete and no longer taken into account in 8.x. If you are using custom index names, please perform the necessary adaptations before attempting to upgrade to 8.x.
+The `kibana.index` and `xpack.tasks.index` configuration settings are obsolete and no longer considered in 9.x. If you use custom index names, please make the necessary adaptations before upgrading to 9.x.
```

Comment:
If there are 8.0-specific upgrade docs somewhere else, then I would simply remove this paragraph.
Otherwise, we can leave the 8.x version numbers and say something like:

> The `kibana.index` and `xpack.tasks.index` configuration settings are obsolete and no longer considered in Kibana 8.x and newer. If you use custom index names, please make the necessary adaptations before upgrading.